app/src/pages/inside/common/launchFiltersToolbar/launchFiltersToolbar.jsx

```diff
@@ -185,9 +193,18 @@ export class LaunchFiltersToolbar extends Component {
     const { dirtyFilterIds, activeFilterId } = this.props;
     return dirtyFilterIds.indexOf(activeFilterId) !== -1;
   };
+  isFilterCreatorCurrentUser = () => {
+    const { userId, activeFilter } = this.props;
+    return activeFilter.owner === userId;
+  };
   isSaveDisabled = () => {
-    const { filterErrors } = this.props;
-    return !this.isFilterUnsaved() || !isEmptyObject(filterErrors) || this.isNoFilterValues();
+    const { filterErrors, isAdmin, userProjectRole } = this.props;
+    return (
+      !this.isFilterUnsaved() ||
+      !isEmptyObject(filterErrors) ||
+      this.isNoFilterValues() ||
+      (!this.isFilterCreatorCurrentUser() && (!isAdmin || userProjectRole !== PROJECT_MANAGER))
```

Comment:
I think it's better to do this check using our permission utils. Please add `EDIT_FILTER` premission to our map.